opensearchpy/_async/client/ingest.py

```diff
@@ -25,42 +25,50 @@
 #  under the License.
 
 
+# ----------------------------------------------------
+# THIS CODE IS GENERATED AND MANUAL EDITS WILL BE LOST.
+#
+# To contribute, kindly make essential modifications through either the "opensearch-py client generator":
+# https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate-api.py
+# or the "OpenSearch API specification" available at:
+# https://github.com/opensearch-project/opensearch-api-specification/blob/main/OpenSearch.openapi.json
+# -----------------------------------------------------
+
+
 from .utils import SKIP_IN_PATH, NamespacedClient, _make_path, query_params
 
 
 class IngestClient(NamespacedClient):
-    @query_params("master_timeout", "cluster_manager_timeout", "summary")
+    @query_params("cluster_manager_timeout", "master_timeout")
     async def get_pipeline(self, id=None, params=None, headers=None):
         """
         Returns a pipeline.
 
 
-        :arg id: Comma separated list of pipeline ids. Wildcards
-            supported
-        :arg master_timeout (Deprecated: use cluster_manager_timeout): Explicit operation timeout for connection
-            to master node
-        :arg cluster_manager_timeout: Explicit operation timeout for connection
-            to cluster_manager node
-        :arg summary: Return pipelines without their definitions
-            (default: false)
+        :arg id: Comma-separated list of pipeline ids. Wildcards
+            supported.
+        :arg cluster_manager_timeout: Operation timeout for connection
+            to cluster-manager node.
+        :arg master_timeout: Operation timeout for connection to master
+            node.
         """
         return await self.transport.perform_request(
             "GET", _make_path("_ingest", "pipeline", id), params=params, headers=headers
         )
 
-    @query_params("master_timeout", "cluster_manager_timeout", "timeout")
+    @query_params("cluster_manager_timeout", "master_timeout", "timeout")
     async def put_pipeline(self, id, body, params=None, headers=None):
         """
         Creates or updates a pipeline.
 
 
-        :arg id: Pipeline ID
+        :arg id: Pipeline ID.
         :arg body: The ingest definition
-        :arg master_timeout (Deprecated: use cluster_manager_timeout): Explicit operation timeout for connection
-            to master node
-        :arg cluster_manager_timeout: Explicit operation timeout for connection
-            to cluster_manager node
-        :arg timeout: Explicit operation timeout
+        :arg cluster_manager_timeout: Operation timeout for connection
+            to cluster-manager node.
+        :arg master_timeout: Operation timeout for connection to master
```

Comment:
May be we could keep the comments for master_timeout that it is deprecated?

Reply:
@VachaShah, Do you mean include deprecation msg in the param description using generator.
something like 




:arg master_timeout (Deprecated: deprecation message from [here](https://github.com/opensearch-project/opensearch-api-specification/blob/main/OpenSearch.openapi.json#L8986) ): Explicit operation timeout for connection
            to master node

Reply:
Yeah! It would be nice to have it in the description of the param.